stubs/extensions/mongodb.phpstub

```diff
@@ -207,44 +235,48 @@ namespace MongoDB\Driver
     final class Cursor implements CursorInterface, Iterator
     {
         /**
-         * @return TValue
+         * @return TValue|null
+         * @psalm-ignore-nullable-return
          */
-        public function current()
+        public function current(): array|object|null
         {
         }
 
-        /**
-         * @return void
-         */
-        public function next()
+        public function next(): void
         {
         }
 
-        /**
-         * @return int
-         */
-        public function key()
+        /** @psalm-ignore-nullable-return */
+        public function key(): ?int
         {
         }
 
-        /**
-         * @return bool
-         */
-        public function valid()
+        public function valid(): bool
         {
         }
 
-        /**
-         * @return void
-         */
-        public function rewind()
+        public function rewind(): void
         {
         }
 
-        /**
-         * @return array<TValue>
-         */
-        public function toArray()
+        /** @return array<TValue> */
+        public function toArray(): array
+        {
+        }
+
+        public function getId(bool $asInt64 = false): Int64|CursorId
```

Comment:
You can improve the return type. The default return type change in 2.0. If we could specify the return type depending on the ext-mongodb version, that would be nice.
```suggestion
        /** @return ($asInt64 is true ? Int64 : CursorId) */
        public function getId(bool $asInt64 = false): Int64|CursorId
```

Reply:
I thought I already had this in, but I didn't add it in our own stubs. It's added now 👍